app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/services/WmsService.scala

```diff
@@ -74,10 +74,13 @@ class WmsService[LayerReader: OgcStore](layers: LayerReader, urlPrefix: String)(
                 .parMapN {
                   case (Valid(mbTile), hists) =>
                     logger.debug(s"Style: ${layer.style}, hists are: ${hists}")
+                    val invisiTile =
+                      mbTile.prototype(params.width, params.height)
+                    val fullTile = invisiTile merge mbTile
```

Comment:
Just piling onto the bandwagon that this is a patch. The problem is in the implementation of `ExtentReification` that `evalExtent` is using. From there this sounds VERY similar to what we encountered during initial development and using `reprojectToRegion` here was critical fix:

https://github.com/geotrellis/geotrellis-server/blob/35bc457021257b7fe7e2f5a82be88d423131a56f/ogc/src/main/scala/geotrellis/server/ogc/OgcLayer.scala#L49-L61

Disclaimer, I'm not clear if that instance is actually being used here, but I think it might be a clue to find root cause.